client/js/restaurants_api.js

```diff
@@ -392,6 +392,7 @@ function mark_selected_value(select_object, value)
 function createNewReview(restaurant_name)
 {
 	var ratingValue = document.forms['editReviewForm']['ratingValue'].value;
+	ratingValue = parseInt(ratingValue);
```

Comment:
this code is not really needed. please use <input type=number> and .valueAsNumber